fastai/transforms.py

```diff
@@ -88,6 +93,7 @@ def googlenet_resize(im, targ, min_area_frac, min_aspect_ratio, max_aspect_ratio
     return out
 
 def cutout(im, n_holes, length):
+    ''' cuts out n_holes number of square holes of size length in image at random locations. holes may be overlapping. '''
```

Comment:
Probably more useful to say 'Implements *cutout* augmentation as described in <URL>' and link to the paper